packages/mikado_reborn/src/components/Progressbar/Progressbar.scss

```diff
@@ -0,0 +1,43 @@
+@import '../../assets/styles/styles.scss';
+
+.mkr__progressbar {
+  display: flex;
+  align-items: center;
+
+  > * + * {
+    margin-left: 1.5rem;
+  }
+
+  &--emoji {
+    height: 1.5rem;
+    display: flex;
+    align-items: center;
+  }
+
+  &--text {
+    @include body-small-font;
+    color: map-get($colors, 'success');
+    font-variant-numeric: tabular-nums;
+  }
+
+  &--bar {
+    background-color: map-get($colors, 'neutral-20');
+    width: 200px;
```

Comment:
The progress bar doesn't make 200px on the mockups.
Maybe it should be able to scale depending on the parent size (so that it can fits more often). If so, we must ensure we have enough space to show the success slot (emoji for now) to prevent the UI from shifting.

Reply:
Oops ! I fixed it with flex-grow to make it fill the space available